tracer/src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -119,6 +119,10 @@
     <DefineConstants>$(DefineConstants);ASYNCLOCAL;HASHTABLE;FEATURE_DEFAULT_INTERFACE;FEATURE_SPAN</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(OS)' == 'Linux' ">
+    <DefineConstants>$(DefineConstants);LINUX</DefineConstants>
+  </PropertyGroup>
```

Comment:
You can't do this - we run the same dlls across all platforms, so you'll need to have a runtime check instead